client/Tutorial/Tutorial.tsx

```diff
@@ -17,47 +19,85 @@ export function Tutorial(props: { onClose: () => void }): JSX.Element {
   }, [props.onClose]);
 
   const advance = () => {
-    if (stepIndex == 0) {
+    if (stepIndex === 0) {
       Metrics.TrackEvent("tutorial_begin");
     }
     const nextStepIndex = stepIndex + 1;
     if (nextStepIndex >= TutorialSteps.length) {
       Metrics.TrackEvent("tutorial_complete");
       return close();
     }
+    setPreviousStepIndex(stepIndex);
     setStepIndex(nextStepIndex);
+    setAwaitActionCompleted(false); 
   };
 
   const step = TutorialSteps[stepIndex];
 
   useEffect(() => {
     const subscription = NotifyTutorialOfAction.subscribe(action => {
       if (step.AwaitAction === action) {
-        advance();
```

Comment:
Because we no longer auto-advance, this can lead to a soft lock in some cases. For instance, if you get to the `ShowInitiativeDialog` step, and the user dismisses the Initiative dialog before continuing to the next step, they'll soft lock on `CompleteInitiativeRolls`.

Reply:
It might be good to add an `AutoAdvance` optional property to TutorialStep, that would apply the hold behavior (immediately advance). This would work for steps like "Click the Characters Tab" or "Click the Roll Initiative button"